src/views.py

```diff
@@ -3,28 +3,16 @@
 
 from flask import (Flask, render_template, flash, request,
                    redirect, url_for, send_from_directory)
-from src.flask_celery import make_celery
-from dotenv import load_dotenv
 
 from src.patent_fetcher import (conduct_search, download_patents,
                                 get_sample_images)
 
 from src.reverse_image_search import identify_results
-
-load_dotenv()
+from src.app import app
```

Comment:
so I put the app instantiation into its own module because we need it in various places